Description: Make cysignals use multiarch paths
 Cysignals uses a very naive way of loading libpython2.7, fix this for Debian
 When we package cysignals for Debian, we will need to move this patch there.
 Also, hey dawg i herd u liek patches
Author: Ximin Luo <infinity0@debian.org>
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- /dev/null
+++ b/sage/build/pkgs/cysignals/patches/multiarch-path.patch
@@ -0,0 +1,10 @@
+--- src/src/scripts/cysignals-CSI	2016-07-07 14:28:52.333831922 +0200
++++ src/src/scripts/cysignals-CSI	2016-07-07 14:27:14.542616423 +0200
+@@ -84,6 +84,7 @@
+     """
+     # Preload the right Python library
+     libpython = os.path.join(sysconfig.get_config_var('exec_prefix'), 'lib',
++                             sysconfig.get_config_var('multiarchsubdir').lstrip('/'),
+                              sysconfig.get_config_var('INSTSONAME'))
+     env = dict(os.environ)
+     if sys.platform == 'macosx':
